src/components/Pager.vue

```diff
@@ -8,7 +8,11 @@
   See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 <template>
-  <div class="pager" role="region" :aria-roledescription="$t('pager.roledescription')">
+  <div
+    :class="['pager', { 'collapsed-controllers': shouldCollapseControllers }]"
```

Comment:
```suggestion
    :class="['pager', { 'with-compact-controls': shouldUseCompactControls }]"
```

For consistency, it might be easier to refer to this state as using "compact controls" instead of adding new terminology for "collapsing controllers" since the div with the controls underneath the pager is already described with the `"compact-controls"` class in the existing codebase.

Reply:
Sure, I replaced the names: https://github.com/apple/swift-docc-render/pull/792/commits/4e0bfd1e312137f17ce3684a4a7c2d29c5ee9d06